src/nodes/victron-nodes.js

```diff
@@ -155,6 +155,22 @@ module.exports = function (RED) {
         }
 
         if (!this.pathObj.disabled && this.service && writepath) {
+
+          // If the value is null, just call. (experimental)
+          // Note: it's not ideal that we set the status before the call to dbus (via this.client.publish()) has happened.
+          // Error handling and feedback might be better, if we wait for the dbus call to return, and then only
+          // set the status. Or better even, set the status to something like "busy..." or "working..." while the
+          // dbus call is happening.
+          if (value === null) {
+            this.client.publish(this.service, writepath, value)
+            this.node.status({
+              fill: 'green',
+              shape,
+              text: 'set to null'
```

Comment:
Prefer to have that with a capital S. Other status texts usually start with a capital letter too.